src/app/components/chat/window/output/item/item.vue

```diff
@@ -15,34 +17,57 @@
 		<div class="chat-window-message-container">
 			<div class="chat-window-message-byline" v-if="!message.combine">
 				<router-link class="chat-window-message-user link-unstyled" :to="message.user.url">
-					{{ message.user.display_name }}
-				</router-link>
-				<span class="chat-window-message-username">@{{ message.user.username }}</span>
-				<span class="chat-window-message-time">
-					<span :title="loggedOn">
+					{{ message.user.display_name }} </router-link
+				><span class="chat-window-message-username"> @{{ message.user.username }}</span
+				><span class="chat-window-message-time">
+					<span v-if="!message._showAsQueued" v-app-tooltip="loggedOn">
 						{{ message.logged_on | date('shortTime') }}
 					</span>
+					<span
+						v-else-if="message._error"
+						class="chat-window-message-byline-error"
+						v-app-tooltip="$gettext(`Failed to send. Press to retry`)"
```

Comment:
I thought about this as well, but I don't think there's much space on mobile. The only thing we could do is pop open a confirm modal on mobile instead of doing the action right away. We should discuss this.